cypress/e2e/rich-text/RichTextEditor.Tables.spec.ts

```diff
@@ -12,10 +12,6 @@ import { RichTextPage } from './RichTextPage';
 describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
   let richText: RichTextPage;
 
-  // copied from the 'is-hotkey' library we use for RichText shortcuts
-  const IS_MAC =
-    typeof window != 'undefined' && /Mac|iPod|iPhone|iPad/.test(window.navigator.platform);
-  const mod = IS_MAC ? 'meta' : 'control';
```

Comment:
It was unused?

Reply:
It is not used anymore, highlighting table content in a different way now as the test was breaking due to the CMD+A logic